exercises/spiral-matrix/description.md

```diff
@@ -0,0 +1,22 @@
+Given a number, return a matrix of numbers in spiral order.
```

Comment:
Maybe `Given a square number N, return a matrix filled with the numbers 1 to N arranged in a spiral`?